### ISSUE TITLE
`requests` not found on `from fastapi.testclient import TestClient`

### ISSUE THREAD

**Issue by nalepae** [tiangolo/fastapi#2595]
### First check

* [X] I added a very descriptive title to this issue.
* [X] I used the GitHub search to find a similar issue and didn't find it.
* [X] I searched the FastAPI documentation, with the integrated search.
* [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [X] I already read and followed all the tutorial in the docs and didn't find an answer.
* [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [X] After submitting this, I commit to one of:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

### How to reproduce the issue

1. Create a new bare python environment (virtual or not)
2. Install `fastapi`: `$ pip install fastapi`
3. Do the following import:
```python
from fastapi.testclient import TestClient
```

### Observed
The following stack trace:
```python
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-37f52789b9f0> in <module>
----> 1 from fastapi.testclient import TestClient

~/.pyenv/versions/3.8.6/envs/test/lib/python3.8/site-packages/fastapi/testclient.py in <module>
----> 1 from starlette.testclient import TestClient as TestClient  # noqa

~/.pyenv/versions/3.8.6/envs/test/lib/python3.8/site-packages/starlette/testclient.py in <module>
     10 from urllib.parse import unquote, urljoin, urlsplit
     11 
---> 12 import requests
     13 
     14 from starlette.types import Message, Receive, Scope, Send

ModuleNotFoundError: No module named 'requests'
```

### Expected
No error

### Workaround
`$ pip install requests`

### Environment
* **OS:** Ubuntu 20.04
* **Python:** 3.8.6 (on virtualenv)
*  **FastAPI:** 0.63.0

**Comment by ArcLightSlavik** [comment]
You can do `pip install fastapi[all]` if you don't want to receive this error, bear in mind that it would also install stuff that you might not need, like jinja templating files setup etc.

**Comment by xaviml** [comment]
Hi @nalepae,

I confirm I see the same as you. However, I believe this is an "issue" from starlette, since you could do the same steps with Starlette:
1. Create a new bare python environment (virtual or not)
1. Install `starlette`: `$ pip install starlette`
1. Do the following import:
```python
from starlette.testclient import TestClient
```

In my opinion, this is expected since `requests` is not needed for production code, just for testing purposes.

Regards,
Xavi M.

**Comment by eboddington** [comment]
I wouldn't say this is entirely expected, but I see that it is documented on [pypi](https://pypi.org/project/fastapi/). I guess one option is to have `fastapi` in our main dependencies and `fastapi[all]` in our dev dependencies, avoiding the need to individually add third party packages?

**Comment by ArcLightSlavik** [comment]
@eboddington That is certainly an option, but I would just add them manually whenever I need them. E.g I get a requests error I add the requests library separately, fastapi[all] installs too many stuff for my liking.

**Comment by eboddington** [comment]
Yeah, I suppose it's not a big deal and for now I've just added `requests` to my dev dependencies to avoid extra bloat. I guess I just found it unexpected initially to use something after installing a package and find that it tried to import something that wasn't in its dependencies.

**Comment by staticdev** [comment]
Still having this on 0.75.0.. would be nice if we had some fix in the imports. Thoughts on that @tiangolo ?

**Comment by briancolfer-upstart** [comment]
Please add this!
